javaslang/generator/Generator.scala

```diff
@@ -68,6 +68,10 @@ def generateMainClasses(): Unit = {
       val JavaStreamType = im.getType("java.util.stream.Stream")
       val TreeSetType = im.getType("javaslang.collection.TreeSet")
       val JavaComparatorType = im.getType("java.util.Comparator")
+      val LinkedHashMapType = im.getType("javaslang.collection.LinkedHashMap")
```

Comment:
Multimaps: Not now, maybe in another PR. They are created with a builder, not with a single constructor or factory method. We need to think about it. Maybe it makes not sense at all...